core/services/chainlink/secret_generator_test.go

```diff
@@ -0,0 +1,28 @@
+package chainlink
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestSecretGenerator_sessionSecret(t *testing.T) {
+	t.Parallel()
+	rootDir := path.Join("/tmp/chainlink_test", "TestSecretGenerator_sessionSecret")
```

Comment:
Is this Okay to hardcode the path here? Why not use `ioutil.TempDir`?

Reply:
Or https://pkg.go.dev/testing#T.TempDir